.gitignore

```diff
@@ -36,3 +36,4 @@ node_modules
 .vendor
 Goopfile.lock
 venv/
+.vscode
```

Comment:
Fixed.
I also changed the return type from action_names to the action itself (see #4010 (last comment))